cli/commands/new_team/__init__.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""Module building the init-team cli command"""
+from pathlib import Path
+
+import click
+
+from cli.commands.common_options import config_option
+from cli.commands.common_options import local_option
+from cli.commands.common_options import verbose_option
+from cli.commands.new_team.new_team import NewTeam
+from cli.obj.config import Config
+
+
+@verbose_option
+@local_option
+@config_option
+@click.option(
+    "--name",
+    "-n",
+    required=True,
+    help="Name of team",
+    type=click.STRING,
+)
+@click.option(
+    "--email",
+    "-e",
+    required=True,
+    help="Email for the team",
```

Comment:
This is less for contact and more to avoid a situation where we have two teams with the same acronym creating a team and being confused.

Say team 1 has the acronym of "abc" and team 2 also has the acronym of "abc". Team 2 would like to see if the team that already exists is actually their team or if it is just a different team with the same acronym, the email is a unique value. A slack channel seems odd as most teams has many, many slack channels (I wouldn't know which one to use for cspi. Would it be forum-interop or forum-we-cspi or forum-qe-layered-product?). That issue is still somewhat relevant with emails, but much less so.

Reply:
Most teams have a team email and even if they used an individual email, it is still a way to differentiate if that team is your team or a different one. We can discuss a different key in our meeting today. I just can’t think of a better one at the moment.